examples/SFVehicles/SF Vehicles/build.gradle

```diff
@@ -38,14 +38,16 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     // You must install or update the Support Repository through the SDK manager to use this dependency.
-    compile 'com.android.support:support-v4:20.+'
+    compile 'com.android.support:support-v4:23.+'
     // You must install or update the Support Repository through the SDK manager to use this dependency.
-    compile 'com.android.support:appcompat-v7:20.+'
+    compile 'com.android.support:appcompat-v7:23.+'
     // You must install or update the Google Repository through the SDK manager to use this dependency.
-    compile 'com.google.android.gms:play-services:6.1.+'
-    // Firebase
-    compile 'com.firebase:firebase-client-android:2.0.+'
+    // Includes Firebase
+    compile 'com.google.android.gms:play-services:9.0.2'
```

Comment:
We should basically never include the target `com.google.android.gms:play-services:x.y.z`, instead narrow it down to the ones we are actually using so we keep the method count low.

The lines below that are commented out are actually what we should be using.